src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -168,7 +168,11 @@ void correspondentExport(LocalDate from, LocalDate to, OutputStreamWriter output
 
             data.forEach((audit) -> {
                 try {
-                    printer.printRecord(parseCorrespondentAuditPayload(audit));
+                    String[] parsedAudit = parseCorrespondentAuditPayload(audit);
```

Comment:
We know the index of caseUuid column here (3) . We could have created a separate method inside exportDataConverter dedicated for caseUUID conversion. The method could take in the index of target column. This would mean we will never try to match UUID that is not caseUUID.

Reply:
Planned for 2nd phase.